src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/event-extension.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/util'], function(util) {
+
+    'use strict';
+
+    var
```

Comment:
This `var` looks a bit lost here... (I know it syntactically correct, but still...)